libs/platform/src/lib/table/components/table-row/table-row.component.ts

```diff
@@ -117,6 +117,10 @@ export class TableRowComponent<T> extends TableRowDirective implements OnInit, A
     @Input()
     isTreeRow: boolean;
 
+    /** Whether the row is only rendering within viewport */
+    @Input()
+    enhancedRendering: boolean;
```

Comment:
Need a more specific name here, maybe "onlyRenderVisibleCells" or something like that?

Reply:
Updated